.github/workflows/gpu-ci.yml

```diff
@@ -1,12 +1,29 @@
-# Sets env vars based on GPU platform
 # Prerequisites
-# - Self-hosted Nvidia GPU runner with `gpu-ci` tag in caller repo
+# - Self-hosted Nvidia GPU runner with CUDA enabled
+# - Runner attached in caller repo with `gpu-ci` label
 # - `cuda` and `opencl` Cargo features
-# - `nextest.toml` `ci` profile and `Cargo.toml` `dev-ci` profile
 name: GPU CI Tests 
 
 on:
+  # We expect dependents to call this on `merge_group` and conditionally run with
+  # if: github.event_name != 'pull_request' || github.event.action == 'enqueued' # 
   workflow_call:
+    inputs:
+      # Nextest profile specified in `.config/nextest.toml`
+      nextest-profile:
+        required: false
+        default: "default"
+        type: string
+      # Cargo profile specified in `Cargo.toml`
+      cargo-profile:
+        required: false
+        default: "release"
```

Comment:
That's not a great idea, as it will strip debug info and make a bunch of stuff unreadable. Us wanting more perf in nova & lurk without giving up on debug info is the reason for those custom profiles.